libs/ledgerjs/packages/cryptoassets/src/currencies.ts

```diff
@@ -2781,18 +2781,24 @@ export const cryptocurrenciesById: Record<CryptoCurrencyId, CryptoCurrency> = {
     managerAppName: "VeChain",
     ticker: "VET",
     scheme: "vechain",
-    color: "#00C2FF",
+    color: "#28008C",
     family: "vechain",
     units: [
       {
         name: "VET",
         code: "VET",
-        magnitude: 8,
+        magnitude: 18,
+      },
+      {
+        name: "WEI",
+        code: "WEI",
+        magnitude: 0,
       },
     ],
     explorerViews: [
       {
-        tx: "https://explore.veforge.com/transactions/$hash",
+        tx: "https://explore-testnet.vechain.org/transactions/$hash",
```

Comment:
We need to remember to update this to point to VeChain stats https://vechainstats.com/
This is not for now but when we change to point to mainnet.

Reply:
perfect